pom.xml

```diff
@@ -1666,6 +1667,14 @@
                 <scope>test</scope>
                 <version>${version.org.wildfly.glow}</version>
             </dependency>
+
+            <!-- JUnit 5 Api -->
+            <dependency>
```

Comment:
Is it possible to revert also changes in top `pom.xml` since no need to use `org.junit.jupiter` ?

Reply:
Fixed in [272066b](https://github.com/keycloak/keycloak/pull/26484/commits/272066b92015fcd415c9f5c5af75fa34e2c66dc5)